.yamllint.yaml

```diff
@@ -0,0 +1,14 @@
+---
+extends: default
+
+ignore: |
+  deploy/helm/**/templates
+
```

Comment:
I think this is leftover from copy/paste

```suggestion
```

Reply:
Yupp, indeed.